Server/Items/SecureTradeContainer.cs

```diff
@@ -106,7 +106,7 @@ public override bool IsChildVisibleTo(Mobile m, Item child)
 		{
 			if (child is VirtualCheck)
 			{
-				return AccountGold.Enabled && (m.NetState == null || !m.NetState.NewSecureTrading);
+				return !AccountGold.Enabled || !m?.NetState?.NewSecureTrading != true;

```

Comment:
I think this probably should be:
```cs
!AccountGold.Enabled || m?.NetState?.NewSecureTrading != true
```

Either Account Gold is disabled, or the client does not support new secure trading.

LMK if that makes sense?